pipeline/scripts/13-excelExport.R

```diff
@@ -381,17 +423,18 @@ if (z_score == 1) {
     colnames(LPI_data) <- c('HMDB.code','HMDB.name','Sample','Zscore')
     
     Pos_Contr <- rbind(PA_data, PKU_data, LPI_data)
-    
-    Pos_Contr <- rbind(PA_data)
-    
-    
+    #Pos_Contr <- rbind(PA_data) #old code, does not add all dataframes together, above is new
     Pos_Contr$Zscore <- as.numeric(Pos_Contr$Zscore)
+    # extra information added to excel for future reference. made in beginning of this script
     Pos_Contr$Matrix <- matrix
     Pos_Contr$Rundate <- rundate
     Pos_Contr$Project <- project
     
     #Save results
-    save(Pos_Contr,file = paste(outdir, 'Pos_Contr.RData', sep = "/"))
+    save(Pos_Contr,file = paste0(outdir, "/", project, '_Pos_Contr.RData'))
+    Pos_Contr$Zscore <- round_df(Pos_Contr$Zscore, 2) # asked by Lab to round the number to 2 digits
+    write.xlsx(Pos_Contr, file = paste0(outdir, "/", project, '_Pos_Contr.xlsx'), sheetName = "Sheet1", col.names = TRUE, row.names = TRUE, append = FALSE)
```

Comment:
Volgens mij kan je dit direct via openxlsx doen. Zie:
https://cran.r-project.org/web/packages/openxlsx/vignettes/Formatting.html
https://stackoverflow.com/questions/49800433/openxlsx-format-cells-as-number-with-comma-and-decimal-places/49800569

Het zou denk ik het netst zijn als het alleen visueel is (in excel)?

Reply:
het is een extra file, naast die files die wel de volledige getalllen hebben. Het was specifiek een wens om het niet visueel te doen, maar echt af te ronden in deze file